Oracle/PCORNetLoader_ora.sql

```diff
@@ -48,85 +48,86 @@ BEGIN
   EXECUTE IMMEDIATE sqlstring;
   dbms_output.put_line(sqlstring);
 END PMN_ExecuateSQL;
-/
 
 
 
 
 
 
-CREATE OR REPLACE SYNONYM I2B2FACT FOR I2B2DEMODATA.OBSERVATION_FACT
-/
+
+CREATE OR REPLACE SYNONYM I2B2FACT FOR I2B2DEMODATA.OBSERVATION_FACT;
 
 
 BEGIN
 PMN_DROPSQL('DROP TABLE i2b2patient_list');
 END;
-/
+
 
 CREATE table i2b2patient_list as 
 select * from
 (
 select DISTINCT PATIENT_NUM from I2B2FACT where START_DATE > to_date('01-Jan-2010','dd-mon-rrrr')
 ) where ROWNUM<100000000
-/
+;
 
 create or replace VIEW i2b2patient as select * from I2B2DEMODATA.PATIENT_DIMENSION where PATIENT_NUM in (select PATIENT_NUM from i2b2patient_list)
-/
+;
 
 create or replace view i2b2visit as select * from I2B2DEMODATA.VISIT_DIMENSION where START_DATE >= to_date('01-Jan-2010','dd-mon-rrrr') and (END_DATE is NULL or END_DATE < CURRENT_DATE) and (START_DATE <CURRENT_DATE)
-/
+;
 
 
 CREATE OR REPLACE SYNONYM pcornet_med FOR  i2b2metadata.pcornet_med
-/
+;
 
 CREATE OR REPLACE SYNONYM pcornet_lab FOR  i2b2metadata.pcornet_lab
-/
+;
 
 CREATE OR REPLACE SYNONYM pcornet_diag FOR  i2b2metadata.pcornet_diag
-/
+;
 
 CREATE OR REPLACE SYNONYM pcornet_demo FOR  i2b2metadata.pcornet_demo
-/
+;
 
 CREATE OR REPLACE SYNONYM pcornet_proc FOR  i2b2metadata.pcornet_proc
-/
+;
 
 CREATE OR REPLACE SYNONYM pcornet_vital FOR  i2b2metadata.pcornet_vital
-/
+;
 
 CREATE OR REPLACE SYNONYM pcornet_enc FOR  i2b2metadata.pcornet_enc
-/
+;
+
 
 create or replace FUNCTION GETDATAMARTID RETURN VARCHAR2 IS 
 BEGIN 
     RETURN 'WF';
 END;
-/
+
 
 CREATE OR REPLACE FUNCTION GETDATAMARTNAME RETURN VARCHAR2 AS 
 BEGIN 
     RETURN 'WakeForest';
 END;
-/
+
 
 CREATE OR REPLACE FUNCTION GETDATAMARTPLATFORM RETURN VARCHAR2 AS 
 BEGIN 
     RETURN '02'; -- 01 is MSSQL, 02 is Oracle
 END;
-/
 
-create or replace view i2b2loyalty_patients as (select patient_num,to_date('01-Jul-2010','dd-mon-rrrr') period_start,to_date('01-Jul-2014','dd-mon-rrrr') period_end from i2b2demodata.loyalty_cohort_patient_summary where BITAND(filter_set, 61511) = 61511 and patient_num in (select patient_num from i2b2patient))
-/
+-- REQUIREMENT: SCILHS-utils/LoyaltyCohort/LoyaltyCohort-ora.sql

```

Comment:
This is good. Of note, Kun didn't bring over my comment section from the beginning of the MSSQL script. Feel free to copy that in too.